xwiki-platform-core/xwiki-platform-livedata/xwiki-platform-livedata-webjar/src/main/webjar/layouts/livedata-table.html

```diff
@@ -1,23 +1,51 @@
 <template>
   <div class="livedata-table-layout">
 
-    <div ref="dropdownMenu"></div>
+    <div ref="topbar"></div>
 
     <!-- Table component -->
     <table class="livedata-table">
 
       <!-- Table Header -->
       <thead>
-        <tr>
 
+        <!-- Column name -->
+        <tr class="column-header">
           <th
             v-for="col in cols"
             :key="col.id"
           >
-            {{ col.name }}
+            <div
+              class="column-name"
+              @click="logic.sort(col.id)"
+            >
+              <span>{{ col.name }}</span>
+              <span
+                :class="[
+                  'sort-icon',
+                  'fa',
+                  { 'sorted': sorts[col.id] },
+                  (sorts[col.id] && sorts[col.id].descending) ? 'fa-caret-up' : 'fa-caret-down',
+                ]"
+              ></span>
+            </div>
           </th>
+        </tr>
 
+        <!-- Column filter -->
+        <tr class="column-filters">
+          <th
+            v-for="col in cols"
+            :key="col.id"
+          >
+            <input
```

Comment:
You need to check if the property is filterable and don't display the input if it's not. I think we said that setting "filter" to false in the property descriptor makes the property unfilterable. We need this for the first version.